.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+##### Summary
```

Comment:
This line is probably not needed as the beginning of PR description would be filled in with commit messages by default...

Reply:
Good point